odl/space/numpy_ntuples.py

```diff
@@ -840,16 +837,23 @@ def _dist(self, x1, x2):
         Examples
         --------
         >>> from numpy.linalg import norm
-        >>> c2_2 = Cn(2, dist=lambda x, y: norm(x - y, ord=2))
+
+        Euclidean distance
```

Comment:
Could be a bit more verbose, complete sentences. This minimum text reads a bit ugly in the formatted doc.